src/collection-preferences/content-display/content-display-option.tsx

```diff
@@ -13,28 +13,22 @@ const componentPrefix = 'content-display-option';
 export const getClassName = (suffix?: string) => styles[[componentPrefix, suffix].filter(Boolean).join('-')];
 
 interface ContentDisplayOptionProps {
-  dragHandleAriaLabel?: string;
-  listeners?: DragHandleProps['listeners'];
+  dragHandleAttributes: DragHandleProps['attributes'];
+  dragHandleListeners?: DragHandleProps['listeners'];
   onToggle?: (option: OptionWithVisibility) => void;
   option: OptionWithVisibility;
-  disabled?: boolean;
```

Comment:
The disabled is replaced with aria-disabled that comes as part of the dragHandleAttributes.